waifus/Hifumi Takimoto.json

```diff
@@ -0,0 +1,13 @@
+{
+	"type" : 0,
+	"english-name" : "Hifumi Takimoto",
+	"image" : "https://vignette.wikia.nocookie.net/new-game/images/9/9d/Hifumi_Game_Render.png",
+	"japanese-name" : "滝本ひふみ",
+	"year" : ¯\_(ツ)_/¯,
```

Comment:
If you already know that someone is legal, you can use the field ``definitely-legal`` and set that to true.

Reply:
I added a quick fix for this. https://github.com/yourWaifu/is-your-waifu-legal/commit/17fa66d8785e35e9af4d0143933c95ab3bd3a335

Reply:
Added "definitely-legal" field